docs/getting_started/install.rst

```diff
@@ -5,6 +5,44 @@ Installation
 Basic Installation
 ------------------
 
+#. Make a directory (folder) for your project.
+#. Create a virtual environment. 
+    **Not sure how to create a virtual environment?**
+    Creating a virtual environment for your project only involves a few commands.
+    See below:
+
+    **Console**
+
+    .. code-block:: console
+
+        $ python -m venv C:\Users\YourComputer\env\name_of_virtual_environment
+        $ python C:\Users\YourComputer\env\name_of_virtual_environment\Scripts\activate.ps1
```

Comment:
New windows and mac sections below look good... you can delete this `**console**` section as it is redundant now.

Reply:
done